src/tests/Common/testenvironment.proj

```diff
@@ -174,6 +174,14 @@
     <TestEnvironment Include="jitstress2_jitstressregs0x10" JitStress="2" JitStressRegs="0x10" />
     <TestEnvironment Include="jitstress2_jitstressregs0x80" JitStress="2" JitStressRegs="0x80" />
     <TestEnvironment Include="jitstress2_jitstressregs0x1000" JitStress="2" JitStressRegs="0x1000" />
+    <!-- There are multiple cases of `jitstress_random_*` so a single pipeline can run random JitStress multiple times without fear
+         of a problem due to repeating the same scenario name. Note that this file is invoked with only one of these at a time.
```

Comment:
not sure if i follow. Why not try different random number for each of the legs?

Reply:
Each leg will get a different random number.

I created `jitstress_random_1` and `jitstress_random_2` because I was worried that YML like:
```
          scenarios:
          - jitstress_random
          - jitstress_random
```
would get confused about the same scenario named twice, even though the goal would be to run the leg twice, each time with a different random number.